Rubrik/Public/Connect-Rubrik.ps1

```diff
@@ -16,7 +16,7 @@ function Connect-Rubrik {
       GitHub: chriswahl
 
       .LINK
-      http://rubrikinc.github.io/rubrik-sdk-for-powershell/reference/Connect-Rubrik.html
+      https://rubrik.gitbook.io/rubrik-sdk-for-powershell/command-documentation/reference/Connect-Rubrik
```

Comment:
@jaapbrasser - none of these cmdlet doc links work. 

The working URL for `Connect-Rubrik` is https://rubrik.gitbook.io/rubrik-sdk-for-powershell/user-documentation-1/workflow-1/connect-rubrik